chat-widget/src/common/telemetry/TelemetryConstants.ts

```diff
@@ -118,6 +118,7 @@ export enum TelemetryEvent {
     InvalidConfiguration = "InvalidConfiguration",
     SendTypingIndicatorSucceeded = "SendTypingIndicatorSucceeded",
     SendTypingIndicatorFailed = "SendTypingIndicatorFailed",
+    event = "event",
```

Comment:
Please update Telemetry.md for this change

Reply:
updated